### DISCUSSION TITLE
Custom Status

### DISCUSSION THREAD

**Discussion by remchuk** [netbox-community/netbox#16916] [Q&A] [answered]
Hey there!

So I want to implement some custom statuses to Devices.

However it looks like it is only done by adding [FIELD_CHOICES](https://github.com/netbox-community/netbox/pull/9041), and to do so means I have to roll out the Netbox.

Is there a way to do it via the UI?

**Comment by candlerb** [comment, marked as answer]
Any settings which are not available at Admin > Configuration History / Config Revisions must be set in `configuration.py`.

I don't know what you mean by "roll out the Netbox". You must have Netbox running somewhere, so you just change the configuration. (Unless you are using a commercial hosted Netbox service, in which case you talk to your provider).

**Reply by remchuk** [reply]
Yes, You are correct, We are running a pod with the Netbox, I just added it to configuration.py and restarted the pod and now everything works! Thanks!

**Reply by candlerb** [reply]
FWIW, if you're running the default netbox-docker image then you can mount a file `configuration/extra.py` which contains extra settings like these, so you don't have to mess with the configuration.py from the image. [Documentation link](https://github.com/netbox-community/netbox-docker/wiki/Configuration#custom-configuration-files)
